helmfile/values/opensearch/dashboards.yaml.gotmpl

```diff
@@ -54,6 +54,9 @@ ingress:
     - host: {{ .Values.opensearch.dashboards.subdomain }}.{{ .Values.global.baseDomain }}
       paths:
         - path: /
+          backend:
+            serviceName:
+            servicePort:
```

Comment:
Why is this needed? Seems like it's incomplete?

Reply:
Was needed to make Helmfile happy!

Reply:
I'd prefer if you actually set them to an empty string if possible so it looks intentional, rather than like something is missing.